src/tree/service.rs

```diff
@@ -95,92 +59,54 @@ impl<M: Middleware> TreeAvailabilityService<M> {
             axum::Server::bind(&addr)
                 .serve(router.into_make_service())
                 .await
-                .map_err(TreeAvailabilityError::HyperError)?;
-            tracing::info!("Server spawned");
-
-            Ok(())
+                .expect("axum server failed");
         });
 
         handles.push(server_handle);
 
-        // Spawn a new task to keep the world tree synced to the chain head
-        handles.push(self.world_tree.spawn());
-
-        handles
+        Ok(handles)
     }
 }
 
 #[derive(Serialize, Deserialize, Debug)]
 #[serde(rename_all = "camelCase", deny_unknown_fields)]
 pub struct InclusionProofRequest {
     pub identity_commitment: Hash,
-    pub root: Option<Hash>,
+    chain_id: Option<ChainId>,
```

Comment:
Add `#[serde(skip_serializing_if = "Option::is_none")]` otherwise we'll always serialize this as `"chainId":null`